chat-plugins/shop.js

```diff
@@ -0,0 +1,145 @@
+/*
+* Credits - Lord Haji, HoeenCoder
+*/
```

Comment:
Well if u see this
Same Comment style is used here too
https://github.com/CreaturePhil/Showdown-Boilerplate/blob/master/chat-plugins/seticon.js